packages/suite/src/components/suite/Settings/components/Theme.tsx

```diff
@@ -46,9 +46,11 @@ const Theme = () => {
     const onChange = ({ value }: typeof THEME_OPTIONS[number]['options'][number]) => {
         if ((value === 'auto') !== autodetectTheme) {
             setAutodetect({ theme: !autodetectTheme });
+            window.desktopApi?.themeSystem();
```

Comment:
The `themeSystem()` method is called even if autodetection is currently turning off.